homework.md

```diff
@@ -4,108 +4,110 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12,240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+CSS is used to give different style to a webpage.  It is used to define styles for web pages, including the design, layout and variations in display for different devices and screen sizes.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
-
+h1 {
+    color: white;
+  text-align: center;
```

Comment:
I think only the # here is necessary to be removed. There is no need adding the styling.

Reply:
Alright Aaishah, I have done the correction.

Thanks for the feedback.